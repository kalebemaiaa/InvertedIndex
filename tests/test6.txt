QueryAndS(QueryOr(QueryWord("chefe"), QueryWord("coordenador")), QueryWord("diplomata"))
11177.raw
5640.raw
980.raw
4678.raw
10968.raw
5439.raw
1090.raw
923.raw
1671.raw
1313.raw
6072.raw
10818.raw
4450.raw
11807.raw
1606.raw
11606.raw
3348.raw
4809.raw
11519.raw
402.raw
91.raw
1453.raw
3285.raw
503.raw
1739.raw
5532.raw
2573.raw
11922.raw
4335.raw
4345.raw
539.raw
517.raw
1152.raw
5613.raw
5925.raw
11892.raw
2145.raw
11472.raw
4130.raw
1086.raw
8792.raw
2151.raw
4108.raw
4867.raw
786.raw
987.raw
2620.raw
278.raw
8886.raw
5145.raw
1305.raw
11230.raw
4231.raw
4965.raw
6008.raw
4495.raw
1639.raw
11426.raw
5012.raw
5575.raw
2075.raw
6154.raw
4918.raw
4294.raw
5823.raw
11568.raw
5755.raw
4034.raw
5651.raw
10974.raw
2412.raw
379.raw
3012.raw
475.raw
2586.raw
11179.raw
3588.raw
4646.raw
2369.raw
10761.raw
6149.raw
2952.raw
5948.raw
3470.raw
11453.raw
3051.raw
2523.raw
4556.raw
672.raw
5995.raw
3246.raw
5088.raw
11811.raw
3066.raw
11987.raw
5284.raw
3706.raw
4668.raw
1840.raw
4410.raw
5843.raw
8719.raw
3829.raw
5590.raw
3420.raw
3284.raw
3067.raw
5973.raw
5263.raw
4234.raw
1828.raw
2228.raw
6165.raw
1659.raw
6009.raw
8834.raw
11310.raw
1907.raw
6386.raw
3179.raw
1646.raw
5408.raw
3282.raw
4063.raw
4368.raw
48.raw
2649.raw
2332.raw
10882.raw
3501.raw
616.raw
2248.raw
1203.raw
3451.raw
1409.raw
2469.raw
5767.raw
1754.raw
2362.raw
1912.raw
6376.raw
4261.raw
5042.raw
6374.raw
1888.raw
3960.raw
1073.raw
3669.raw
911.raw
1252.raw
11303.raw
1628.raw
5182.raw
2851.raw
3459.raw
3956.raw
5468.raw
162.raw
3367.raw
4634.raw
5121.raw
3475.raw
316.raw
5730.raw
10796.raw
1136.raw
2020.raw
1480.raw
2836.raw
4974.raw
1642.raw
2109.raw
4769.raw
4153.raw
2575.raw
3345.raw
3943.raw
3776.raw
2388.raw
2992.raw
5374.raw
392.raw
2290.raw
2447.raw
5054.raw
3200.raw
1847.raw
2111.raw
1701.raw
1546.raw
11565.raw
5191.raw
1263.raw
2178.raw
11557.raw
5351.raw
3811.raw
932.raw
5498.raw
4961.raw
