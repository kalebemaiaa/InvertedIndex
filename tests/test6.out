2053
11570
